File: PRAK205-2510817220023-NURFITRIA RAMADHANI.c
#include<stdio.h>
#include<math.h>

float alas(float a, float b){
float c;
    c = sqrt(pow(b,2)-pow(a,2));
    return c;
}

float keliling(float a, float b, float c){
float k = a+b+c;
    return k;
}

float luas(float a, float c){
float l = (a*c)/2;
    return l;
}

int main(){
float a, b, c, k, l;

    scanf("%g", &a);
    scanf("%g", &b);
    
    c = alas(a,b);
    k = keliling(a,b,c);
    l = luas(a,c);
    
    
    printf("Alas = %g cm\n", c);
    printf("Tinggi = %g cm\n", a);
    printf("Keliling = %g cm\n", k);
    printf("Luas = %g cm^2\n", l);

    return 0;
}

File: PRAK205-2510817220023-NURFITRIA RAMADHANI.py
import math

def alas(a,b):
    c = math.sqrt((b**2)-(a**2))
    return c
    
def keliling(a,b,c):
    k = a+b+c
    return k
    
def luas(a,c):
    l = (a*c)/2
    return l
    
a,b = map(float,input().split())
c = alas(a,b)  
print(f"Alas = {alas(a,b):.0f} cm")
print(f"Tinggi {a:.0f} cm")
print(f"Keliling = {keliling(a,b,c):.0f} cm")
print(f"Luas = {luas(a,c):.0f} cm^2")
print()
a = float(input())
b = float(input())
c = alas(a,b)  
print(f"Alas = {alas(a,b):.0f} cm")
print(f"Tinggi {a:.0f} cm")
print(f"Keliling = {keliling(a,b,c):.0f} cm")
print(f"Luas = {luas(a,c):.0f} cm^2")